### PR TITLE
Added more R toolbox suggests.

### PR DESCRIPTION
**Pull request by wendycwong**
My 2d pdp requires more R toolboxes.  I forgot to add them into the correct places.

### CODE REVIEW

File: h2o-r/h2o-DESCRIPTION.template [hunk 1]
```diff
@@ -55,6 +55,10 @@ Suggests: ggplot2,
           slam,
           bit64 (>= 0.9.7),
           data.table (>= 1.9.8),
+          open3d,
+          persp3D,
+          persp3Drgl,
+          snapshot3d,
```
Comment (ledell) [inline h2o-r/h2o-DESCRIPTION.template]:
```suggestion
```

Comment (ledell) [reply h2o-r/h2o-DESCRIPTION.template]: This is a function name, not a library name.  The library is `persp3Drgl`.

File: h2o-r/h2o-DESCRIPTION.template [hunk 2]
```diff
@@ -55,6 +55,10 @@ Suggests: ggplot2,
           slam,
           bit64 (>= 0.9.7),
           data.table (>= 1.9.8),
+          open3d,
```
Comment (ledell) [inline h2o-r/h2o-DESCRIPTION.template]:
```suggestion
```

Comment (ledell) [reply h2o-r/h2o-DESCRIPTION.template]: This is a function name, not a library name. The library is `persp3D`.

Comment (wendycwong) [reply h2o-r/h2o-DESCRIPTION.template]: Erin:  

I removed all the packages and only keep rgl, plot3Drgl.

**Comment by ledell** [comment]
@wendycwong Ok, this is good now!

File: h2o-r/h2o-DESCRIPTION.template [hunk 3]
```diff
@@ -55,6 +55,8 @@ Suggests: ggplot2,
           slam,
           bit64 (>= 0.9.7),
           data.table (>= 1.9.8),
+          rgl,
```
Comment (michalkurka) [inline h2o-r/h2o-DESCRIPTION.template]: I think it is better to be on a safe side here and specify the version that is being tested as the minimal version

File: h2o-r/h2o-package/R/models.R [hunk 4]
```diff
@@ -3473,8 +3473,6 @@ row_index=-1) {
   if (!noPairs &&
       ("plot3Drgl" %in% rownames(installed.packages())) &&
       ("rgl" %in% rownames(installed.packages()))) {
```
Comment (michalkurka) [inline h2o-r/h2o-package/R/models.R]: please generate a warning when either of these packages is not available. the warning should instruct users what specifically they need to do in order to enable this functionality

Comment (wendycwong) [reply h2o-r/h2o-package/R/models.R]: Done.

File: docker/Makefile [hunk 5]
```diff
@@ -16,7 +16,7 @@ else
 endif
 
 dev-base:
-	docker build -t harbor.h2o.ai/opsh2oai/h2o-3/dev-base$(GPU_SUFFIX):$(VERSION) \
+	docker build --no-cache -t harbor.h2o.ai/opsh2oai/h2o-3/dev-base$(GPU_SUFFIX):$(VERSION) \
```
Comment (michal-raska) [inline docker/Makefile]: I'm not sure we want `--no-cache`. For example it takes very long to build the R images. Why do you need it please?

Comment (honzasterba) [reply docker/Makefile]: I had a suspicion that a change I made did not propagate into the R images. Reverted.

**Pull request merged**